cmd/repository/default-pkg-repository.go

```diff
@@ -15,16 +15,17 @@ const (
 
 type defaultPackageRepository struct {
 	RepoDir  string
```

Comment:
We can put this into the interface. A package repository has always a repository directory